Lib/pprint.py

```diff
@@ -342,6 +342,33 @@ def _pprint_mappingproxy(self, object, stream, indent, allowance, context, level
 
     _dispatch[_types.MappingProxyType.__repr__] = _pprint_mappingproxy
 
+    def _pprint_simplenamespace(self, object, stream, indent, allowance, context, level):
+        if type(object) is _types.SimpleNamespace:
+            # the SimpleNamespace repr is "namespace", while the class name is
```

Comment:
Thanks for adding a comment here.  That helps. :)  FWIW, the following might be more clear:

```
# The SimpleNamespace repr has "namespace" instead of the class name, so
# we do the same here.  For subclasses we just use the class name.
```

Reply:
Tried improving it, should be clear now